jadepix1/python/cceanalysis.py

```diff
@@ -8,83 +8,126 @@
 __copyright__ = "Copyright (c) yangtao"
 __created__ = "[2018-03-09 Fri 19:00]"
 
-import sys,os,re
+import sys,os
+import re
+import string
 import numpy as np
 import ROOT
 import logging
+import ConfigParser
 
 class cceanalysis():
```

Comment:
Please rename 'cceanalysis' according to PEP8 (e.g. 'CCEAnalysis')  

https://www.python.org/dev/peps/pep-0008/